pkgs/servers/sql/postgresql/ext/timescaledb.nix

```
@@ -12,8 +12,8 @@
 # }
 
 stdenv.mkDerivation rec {
-  pname = "timescaledb";
-  version = "2.10.0";
+  pname = "timescaledb${lib.optionalString (!enableUnfree) "-apache"}";
+  version = "2.10.1";
```

```suggestion
  version = "2.10.0";
```

Please undo that version bump here, as it has been merged already.
See https://github.com/NixOS/nixpkgs/pull/220264

Plus, it would require the hash further down below to change too, just like in the linked PR.
You can, however, if you'd like, rebase your PR for it, but you do not need to.

---

Oh lol, you hit merge seconds before I send that review :eyes: 
Sorry^^

Can you please rebase instead? :)

---

Ouch, obviously it affects the hash value 🤦‍♂️ I'll quickly rebase it.

---

Done, sorry again. That one was stupid 😓